SETUP.md

```diff
@@ -139,24 +136,26 @@ SPARK_WORKER_OPTS="-Dspark.worker.cleanup.enabled=true, -Dspark.worker.cleanup.a
 
 ## Setup guide for Azure Databricks
 
-### Requirements of Azure Databricks
+### Requirements
 
-* Runtime version 4.3 (Apache Spark 2.3.1, Scala 2.11)
+* Databricks Runtime version 4.3 (Apache Spark 2.3.1, Scala 2.11) or greater
```

Comment:
Not related to this PR but CosmosDB now supports Spark 2.4. We should update our reqs to Databricks 5.0 - see #669.